src/context/WalletProvider/NativeWallet/components/LegacyTwoFactor.tsx

```diff
@@ -0,0 +1,69 @@
+import {
+  Button,
+  FormControl,
+  FormErrorMessage,
+  Input,
+  ModalBody,
+  ModalHeader,
+} from '@chakra-ui/react'
+import { FieldValues, useForm } from 'react-hook-form'
+import { useTranslate } from 'react-polyglot'
+import { RouteComponentProps } from 'react-router-dom'
+import { Text } from 'components/Text'
+
+// @TODO(NeOMakinG): Change this with the mnemonic of the legacy account
+const DUMMY_MNEMONIC = 'yolo yolo yolo yolo yolo yolo yolo yolo yolo yolo yolo yolo'
+
+export const LegacyTwoFactor = ({ history }: RouteComponentProps) => {
+  const {
+    handleSubmit,
+    register,
+    reset,
+    formState: { errors, isSubmitting },
+  } = useForm({ shouldUnregister: true })
+
+  const translate = useTranslate()
+
+  const onSubmit = async (values: FieldValues) => {
+    history.push('/native/legacy/login/success', { mnemonic: DUMMY_MNEMONIC })
+    // Reset the field in order to remove the 2fa code from the memory
+    reset()
+    return
```

Comment:
Again, prepare for async call.

```suggestion
    try { 
      await new Promise((resolve, reject) => resolve())
      // TODO use response to create a new vault
      history.push('/native/legacy/login/success', { vault })
      // Reset the field in order to remove the 2fa code from the memory
      reset()
      return
    } catch (err) {
      // TODO: show 2fa error
    }
```

Reply:
I guess we don't need this if we merge the 2fa into the login modal